__tests__/app.test.js

```diff
@@ -191,3 +191,28 @@ describe("/api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("status 200 return article array ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(12);
+        expect(articles).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              comment_count: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            }),
+          ])
+        );
```

Comment:
`the articles should be sorted by date in descending order.`
^ from trello

I'd like to see some testing here to check if the default order is what we expect it to be